library/camel-kamelets-catalog/src/main/resources/kamelets/jira-add-comment-sink.kamelet.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: camel.apache.org/v1alpha1
+kind: Kamelet
+metadata:
+  name: jira-add-comment-sink
+  annotations:
+    camel.apache.org/catalog.version: "kamelet-catalog-1.9-SNAPSHOT"
```

Comment:
1.8

Reply:
Ah, forgot to push after I changed to 1.8. Thanks.